README.md

```diff
@@ -72,6 +72,28 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```

Comment:
- [ ] You've done well by adding key feature as requested, but please kindly remove this template that you mistakenly added alongside (I guess)  so as to make your README file look professional. 👍 

> **Note: The same thing is applicable to `line 91` and `line 174`**.